src/api/app/views/webui/request/show.html.erb

```diff
@@ -50,11 +50,16 @@
         <% end %>
         <% if @accept_at -%>
             <li><%= sprite_tag('exclamation', title: '') %>
-               <% if Time.now > Time.parse(@accept_at.to_s) %>
-                 This request will get accepted automatically when it enters the 'new' state!
+              <% if BsRequest::FINAL_REQUEST_STATES.include?(@state) %>
+                The request auto-accept was set to
+                <span title='<%= @accept_at.utc.strftime('%Y-%m-%d %H:%M UTC') %>' class='fuzzy-time'>
+                  <%= @accept_at.utc.strftime('%Y-%m-%d') %>.
+                </span>
+              <% elsif @accept_at.past? %>
+                This request will get accepted automatically when it enters the 'new' state!
```

Comment:
[Nitpick]

Slightly better:
"The request will be automatically accepted  when it enters the 'new' state." IMO we don't need an exclamation mark.